### ISSUE TITLE
use torchDynamo instead of tracing to get graph?

### ISSUE THREAD

**Issue by Jack47** [pytorch/tau#434]
hi guys, this project is awesome comparing with torch pipe, I have some models that are [not supported by tracing](https://pytorch.org/docs/stable/fx.html#limitations-of-symbolic-tracing). Would you guys have plan to support torchDynamo to get fx.graph?

**Comment by anj-s** [comment]
Yes, definitely. We do plan to support torchdynamo as the frontend in the near future.

**Comment by Jack47** [comment]
> Yes, definitely. We do plan to support torchdynamo as the frontend in the near future.

cool, glad to hear that, I will test it once released!

**Comment by Vatshank** [comment]
Hi @anj-s, is TorchDynamo support being worked on? I see dynamo mentioned in some of the code under `spmd/compiler` and `pippy/fx`but wasn't sure if the integration is a work in process or completed? 

Happy to try contributing any missing pieces if you think that will be helpful!
